openmdao/core/problem.py

```diff
@@ -583,11 +586,100 @@ def check_partials(self, out_stream=sys.stdout, comps=None, compact_print=False,
         # Conversion of defaultdict to dicts
         partials_data = {comp_name: dict(outer) for comp_name, outer in iteritems(partials_data)}
 
-        _assemble_derivative_data(partials_data, rel_err_tol, abs_err_tol, out_stream,
+        logging.getLogger().setLevel(logging.INFO)
+        _assemble_derivative_data(partials_data, rel_err_tol, abs_err_tol, logger,
                                   compact_print, comps, global_options)
 
         return partials_data
 
+    def check_total_derivatives(self, of=None, wrt=None, logger=None, compact_print=False,
```

Comment:
You should edit the README.md file and remove 'Total Derivatives checking' from the list of things we don't support.